docs/source/conf.py

```diff
@@ -187,4 +187,4 @@
 
 
 def setup(app):
-    app.add_stylesheet('params.css')
+    app.add_css_file('params.css')
```

Comment:
I'm surprised this is needed... maybe version conflict?
The canonical docs build happens through ops so we should check with @dillon-cullinan and see if this new API is expected.

Reply:
I would just ping @dillon-cullinan and make sure he's cool with this, since I think he knows more about docs builds than I do.

Reply:
Thumbs up from Dillon on this.